src/interaction/InteractionManager.js

```diff
@@ -341,6 +459,56 @@ InteractionManager.prototype.dispatchEvent = function ( displayObject, eventStri
 };
 
 /**
+ * If pointer events aren't available on a device, this will turn either the touch or mouse events into pointer events
+ * This allows a developer to just listen for emitted pointer events on interactive sprites
+ *
+ * @param eventString {string} the name of the event (e.g, mousedown)
+ * @private
+ */
+InteractionManager.prototype.normalizeEventString = function ( eventString )
+{
+    if (!this.supportsPointerEvents)
+    {
+        if (this.supportsTouchEvents && Device.any)
```

Comment:
Replied below. Device.any is essentially 'any mobile device'. It can't be _any_ device that supports touch, because that would cover using Chrome on a touch screen laptop... and then any mouse/trackpad clicks wouldn't work! mouse events are still triggered when the screen is touched in this scenario however, so you're not losing that functionality.